db/seeds.rb

```diff
@@ -5,3 +5,18 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+# Shelter.destroy_all
+# Pet.destroy_all
+# Application.destroy_all
```

Comment:
You'll need to destroy your seeds in order to prevent duplicates, just make sure you're careful about the order in which you're destroying them (or check out `dependent: :destroy`

For example, you can't destroy Shelters before you destroy Pets because Pets have a foreign key to a shelter. So after you run line 9, there are pets in the database with invalid foreign keys to shelters. So you need to delete any tables with foreign keys (think child tables or joins tables) **before** you delete any parent tables.